vmdb/app/models/condition.rb

```diff
@@ -117,7 +117,7 @@ def self.subst(expr, rec, inputs)
   end
 
   def self._subst(rec, inputs, opts, tag, mode)
-    ohash, ref, object = self.options2hash(opts, rec)
+    ohash, ref, _ = self.options2hash(opts, rec)
```

Comment:
Instead of blanking out the variable name entirely can we use `_object`?  The reason I ask is that it makes it clear what is being returned, if used later, without having to actually save it into a real variable.